arch/arm/arm64/include/uk/asm/paging.h

```diff
@@ -285,12 +285,13 @@ static inline int ukarch_pt_write_base(__paddr_t pt_paddr)
 
 static inline void ukarch_tlb_flush_entry(__vaddr_t vaddr)
 {
+	__u64 page_number = vaddr >> PAGE_SHIFT;
```

Comment:
I would prefer term VPN (virtual page number). page_number usually is something associated with the physical page

Reply:
The names that I am used to are _pages for the virtual memory_ and _frames for physical memory_. Anyways, renaming it to `virtual_page_number` is also fine to me.

Maybe there is some canonical name for that number in Unikraft?

Reply:
I agree with @kubanrob, "virtual" seems redundant since we are referring to a page already. I'm okay to keep it as is. What caught my eye instead is shifting by `PAGE_SIZE`, which in theory is configurable between the standard 4KiB to 64KiB. However, the architecture takes care of that. From the Arm ARM:

> The treatment of the low-order bits of this field depends on the translation granule size, as follows:
> • Where a 4KB translation granule is being used, all bits are valid and used for the invalidation.
> • Where a 16KB translation granule is being used, bits [1:0] of this field are RES0 and ignored
when the instruction is executed, because VA[13:12] have no effect on the operation of the
instruction.
> • Where a 64KB translation granule is being used, bits [3:0] of this field are RES0 and ignored
when the instruction is executed, because VA[15:12] have no effect on the operation of the
instruction.

So I think it's all good 👍🏼